modules/core/src/main/java/org/apache/ignite/spi/systemview/view/TransactionView.java

```diff
@@ -0,0 +1,190 @@
+    }
+
+    /** @return Start time of this transaction on this node. */
+    @Order(4)
+    public long startTime() {
+        return tx.startTime();
+    }
+
+    /** @return Isolation level. */
+    @Order(5)
+    public TransactionIsolation isolation() {
+        return tx.isolation();
+    }
+
+    /** @return Concurrency mode. */
+    @Order(6)
+    public TransactionConcurrency concurrency() {
+        return tx.concurrency();
+    }
+
+    /** @return Current transaction state. */
+    @Order(1)
+    public TransactionState state() {
+        return tx.state();
+    }
+
+    /** @return Transaction timeout value. */
+    public long timeout() {
+        return tx.timeout();
+    }
+
+    /** @return {@code True} if transaction was started implicitly. */
+    public boolean implicit() {
+        return tx.implicit();
+    }
+
+    /** @return Transaction UID. */
+    @Order(2)
+    public IgniteUuid xid() {
+        return tx.xid();
+    }
+
+    /** @return {@code True} if transaction is started for system cache. */
+    public boolean system() {
+        return tx.system();
+    }
+
+    /** @return Flag indicating whether transaction is implicit with only one key. */
+    public boolean implicitSingle() {
+        return tx.implicitSingle();
+    }
+
+    /** @return {@code True} if near transaction. */
+    public boolean near() {
+        return tx.near();
+    }
+
+    /** @return {@code True} if DHT transaction. */
+    public boolean dht() {
+        return tx.dht();
+    }
+
+    /** @return {@code True} if dht colocated transaction. */
+    public boolean colocated() {
+        return tx.colocated();
+    }
+
+    /** @return {@code True} if transaction is local, {@code false} if it's remote. */
+    public boolean local() {
+        return tx.local();
+    }
+
+    /** @return Subject ID initiated this transaction. */
+    public UUID subjectId() {
+        return tx.subjectId();
+    }
+
+    /** @return Label of transaction or {@code null} if there was not set. */
+    @Order(3)
+    public String label() {
+        return tx.label();
+    }
+
+    /** @return {@code True} if transaction is a one-phase-commit transaction. */
+    public boolean onePhaseCommit() {
+        return tx.onePhaseCommit();
+    }
+
+    /** @return {@code True} if transaction has at least one internal entry. */
+    public boolean internal() {
+        return tx.internal();
+    }
+
+    /** @return Originating node id. */
+    @Order
+    public UUID originatingNodeId() {
+        return tx.originatingNodeId();
+    }
+
+    /** @return Other node id. */
+    public UUID otherNodeId() {
+        return tx.otherNodeId();
+    }
+
+    /** @return Topology version. */
+    public String topVer() {
+        return Objects.toString(tx.topologyVersion());
+    }
+
+    /** @return Duration in millis. */
+    public long duration() {
+        return U.currentTimeMillis() - tx.startTime();
+    }
+
+    /** @return Keys count. */
```

Comment:
It's not clear for me what does the "Keys count" mean? Could you please, give more descriptive JavaDoc.